rbi/stdlib/rdoc.rbi

```diff
@@ -8145,7 +8145,7 @@ class RDoc::RDoc
 
   # Removes file extensions known to be unparsable from `files` and TAGS files
   # for emacs and vim.
-  def remove_unparseable(files); end
+  def remove_unparsable(files); end
```

Comment:
The files in `rbi/` declare methods provided by either the Ruby standard library or common gems. In the rdoc gem, the name of this method is actually `remove_unparseable`

https://ruby.github.io/rdoc/RDoc/RDoc.html#method-i-remove_unparseable

Would you mind going through and either getting rid of the changes in `rbi/` or confirming that there actually was a mismatch between the method that the corresponding gem declares and the RBI file that Sorbet ships with?

Reply:
Absolutely. This was the type of feedback I was hoping to get, so thank you for that. I'll run through and make the appropriate changes.

Reply:
I've confirmed all the `rbi/` issues are indeed upstream, so I've dropped those 3 commits.